.github/workflows/tests-polkadot-deps.yml

```diff
@@ -77,10 +86,10 @@ jobs:
 
     strategy:
       matrix:
-        image: ['latest-rc']
-        required: ['optional']
+        image: ['latest']
```

Comment:
latest-master required and latest-develop optional?

Reply:
waiting for answer regarding node tagging

Reply:
so, the `latest` tag moves forward with every release, so I think this is fine

Reply:
Ok I’ll merge then !